### PR TITLE
Backend: Job magic params

### PR DESCRIPTION
**Pull request by EdenComp**
Also fixes ARE-351

### CODE REVIEW

File: backend/back/src/migrations/1698521602687-WorkflowAreaReturnParams.ts [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+		},
+		{
+			areas: ["seconds-interval"],
+			parameters: ["seconds"],
+		},
+	];
+
+	public async up(queryRunner: QueryRunner): Promise<void> {
+		await queryRunner.query(`
+            alter table area
```
Comment (RezaRahemtola) [inline backend/back/src/migrations/1698521602687-WorkflowAreaReturnParams.ts:38]: lowercase SQL vibes

File: backend/workers/google/src/docs/create_document.py [hunk 2]
```diff
@@ -36,7 +36,6 @@ def create_document():
             params = Struct()
             params.update({
                 "workflowStepId": args["workflowStepId"],
-                "docTitle": document.get('title')
```
Comment (RezaRahemtola) [inline backend/workers/google/src/docs/create_document.py:39]: why did you remove this?

File: backend/workers/google/src/youtube/on_video.py [hunk 3]
```diff
@@ -63,8 +62,6 @@ def on_video():
                 with grpc.insecure_channel(target) as channel:
                     params = Struct()
                     params.update({
-                        "channelId": last_video["channelId"],
-                        "videoId": last_video["videoId"],
```
Comment (RezaRahemtola) [inline backend/workers/google/src/youtube/on_video.py:67]: same

File: backend/back/src/migrations/1698521602687-WorkflowAreaReturnParams.ts (hunk 1 above)
Comment (Croos3r) [reply backend/back/src/migrations/1698521602687-WorkflowAreaReturnParams.ts:38]: ssshhhhhh, you are talking too loud, use sql not the uppercase version (⚠ too loud ⚠)

File: backend/back/src/migrations/1698521602687-WorkflowAreaReturnParams.ts [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+                add column parameters_return_flow text[] not null default array[]::text[];
+        `);
+
+		for (const params of this.areaParametersReturnFlow) {
+			await queryRunner.query(`
+                update area
+                set parameters_return_flow = '{${params.parameters.map((v) => `"${v}"`).join(",")}}'
```
Comment (Croos3r) [inline backend/back/src/migrations/1698521602687-WorkflowAreaReturnParams.ts:45]: Don't you need a JSON.stringify somewhere over here ?

File: backend/back/src/services/entities/area.entity.ts [hunk 5]
```diff
@@ -28,6 +28,10 @@ export default class Area {
 	@Column({ type: "jsonb", default: [] })
 	parametersFormFlow!: ParametersFormFlowFieldDto[];
 
+	@ApiProperty()
+	@Column({ array: true, type: "text", default: [] })
+	parametersReturnFlow!: string[];
```
Comment (Croos3r) [inline backend/back/src/services/entities/area.entity.ts:33]: What this is going to do ?

**Review by Croos3r — CHANGES_REQUESTED** [review]

File: backend/workers/google/src/docs/create_document.py (hunk 2 above)
Comment (EdenComp) [reply backend/workers/google/src/docs/create_document.py:39]: Data wasn't relevant for the user, I think we shouldn't expose it unless we could provide a link for it or something usable for the final user

File: backend/workers/google/src/youtube/on_video.py (hunk 3 above)
Comment (EdenComp) [reply backend/workers/google/src/youtube/on_video.py:67]: Same

File: backend/back/src/services/entities/area.entity.ts (hunk 5 above)
Comment (EdenComp) [reply backend/back/src/services/entities/area.entity.ts:33]: This array in db is for the front, who will show the magic params available for the area

File: backend/back/src/migrations/1698521602687-WorkflowAreaReturnParams.ts (hunk 4 above)
Comment (EdenComp) [reply backend/back/src/migrations/1698521602687-WorkflowAreaReturnParams.ts:45]: JSON.stringify() uses braces [] instead of brackets {} which are needed for postgres arrays

File: backend/back/src/services/entities/area.entity.ts (hunk 5 above)
Comment (Croos3r) [reply backend/back/src/services/entities/area.entity.ts:33]: And what are the magic params exactly ? It's not very self-explanatory

File: backend/back/src/migrations/1698521602687-WorkflowAreaReturnParams.ts (hunk 4 above)
Comment (Croos3r) [reply backend/back/src/migrations/1698521602687-WorkflowAreaReturnParams.ts:45]: Ok okay, tought it was jsonb

**Review by RezaRahemtola — APPROVED** [review]

**Review by Croos3r — APPROVED** [review]
LGTM
